src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -527,3 +534,27 @@ experimental=@DEFAULTEXPFEATURES@
 # If enabled, user can run pprof tools with shim v2 process through kata-monitor.
 # (default: false)
 # enable_pprof = true
+
+[image]
+# Container image service.
+#
+# Offload the CRI image management service to the kata agent.
+service_offload = true
```

Comment:
I find the `service_offload` name to be too generic. What about `guest_image_download` instead?

Reply:
Do we want to specify here if we want only encrypted images even if we run a non-protected container? In other words, do we need an "encryption" setting for guest image download?

Reply:
we do,  since it can work with both (protected and non-protected) containers and afaik it can be optional

cc @sameo

Reply:
> I find the `service_offload` name to be too generic. What about `guest_image_download` instead?

It's not about downloading the guest image, but the container image(s). And the namespaced path for that knob would be `image.service_offload`, which is less generic.

Reply:
@c3d  wdyt? ^^^

Reply:
ok, no response for more than two weeks, I will consider this as resolved